src/pages/patientView/sampleManager.tsx

```diff
@@ -180,10 +180,23 @@ class SampleManager {
 
     }
 
-    getPopoverSample(sample: ClinicalDataBySampleId, sampleNumber: number) {
+    getPopoverSample(sample: ClinicalDataBySampleId, sampleNumber: number, sampleColor:string, fillOpacity: number, extraText: string) {
+
         return (
             <div style={{ maxHeight:400, overflow:'auto' }}>
-                <h5>{ sample.id }</h5>
+                <h5 style={{ marginBottom: 1 }}>
+                    <svg height="12" width="12" style={{ marginRight: 5}}>
+                        <SampleInline
+                                     sample={sample}
+                                     sampleNumber={sampleNumber}
+                                     sampleColor={sampleColor}
+                                     fillOpacity={fillOpacity}
+                                 >
```

Comment:
Indentation looks off here. Also, lint doesn't like empty bodies so it might be better to do 
```
<SampleInline
    sample={sample}
    ...
/>
```